src/addons/par/icc/rules.mk

```diff
@@ -75,4 +75,4 @@ $(OBJ_PAR_DIR)/generated/$(PAR_TOP).macros.out: $(OBJ_PAR_DIR)/synopsys-icc-macr
 $(OBJ_PAR_DIR)/generated/$(PAR_TOP).floorplan.json: \
 		$(PAR_TOOL_ADDON)/tools/generate-floorplan-json \
 		$(OBJ_PAR_DIR)/generated/$(PAR_TOP).macros.out
-	$(abspath $<) --macros $(filter %.macros.out,$^) --rtl_top $(SYN_TOP) --config $(PAR_CONFIG) -o $@
+	PATH="$(abspath $(dir $(CMD_PYTHON3))):$(PATH)" $(abspath $<) --macros $(filter %.macros.out,$^) --rtl_top $(SYN_TOP) --config $(PAR_CONFIG) -o $@
```

Comment:
I agree; the main annoying issue, however, is that Palmer set up PLSI to build CMD_PYTHON3 as a dependency in the rules (i.e. it builds python3 from source). Making it global would require me to refactor the way PLSI handles these kinds of binary/tool dependencies, which is something that is on my TODO list and will be handled in a future commit. Let me know what you think.